### PR TITLE
Onboard OCM repo of stolostron.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Epic: https://issues.redhat.com/browse/ACM-5551

### CODE REVIEW

**Comment by xuezhaojun** [comment]
/pj-rehearse pull-ci-stolostron-registration-operator-backplane-2.4-e2e

File: ci-operator/config/stolostron/addon-framework/stolostron-addon-framework-backplane-2.4.yaml [hunk 1]
```diff
@@ -28,9 +28,6 @@ images:
       as:
       - golang:1.19
   to: addon-examples
-promotion:
-  name: "2.4"
```
Comment (xuezhaojun) [inline ci-operator/config/stolostron/addon-framework/stolostron-addon-framework-backplane-2.4.yaml:32]: Addon-manager is also moved into ocm repo. @gurnben

**Review by gurnben — APPROVED** [review]
/lgtm
/approve

**Pull request merged**